### PR TITLE
RayContext init return address info

### PR DESCRIPTION
**Pull request by hkvision**
Return plasma address, etc.
Same as ray.init(), it returns such info.

### CODE REVIEW

**Comment by hkvision** [comment]
jenkins passed: http://10.239.47.210:18888/job/ZOO-PR-Validation/3464/

File: pyzoo/zoo/ray/raycontext.py [hunk 1]
```diff
@@ -308,12 +308,12 @@ def init(self, driver_cores=0):
             if self.env:
                 os.environ.update(self.env)
             import ray
-            ray.init(num_cpus=self.ray_node_cpu_cores,
-                     object_store_memory=self.object_store_memory,
-                     resources=self.extra_params)
+            return ray.init(num_cpus=self.ray_node_cpu_cores,
```
Comment (jason-dai) [inline pyzoo/zoo/ray/raycontext.py:311]: instead return this to user, store the info in RayContext; the user can the call RayContext methods to get relevant information.

Comment (hkvision) [reply pyzoo/zoo/ray/raycontext.py:311]: What about both return and store? This aligns with ray as it ray init does return something...

Comment (jason-dai) [reply pyzoo/zoo/ray/raycontext.py:311]: ok

**Comment by hkvision** [comment]
jenkins passed: http://10.239.47.210:18888/job/ZOO-PR-Validation/3494/

**Pull request merged**